account_payment_purchase/tests/test_account_payment_purchase.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2013-2015 Tecnativa - Pedro M. Baeza
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import fields
+from odoo.tests import TransactionCase, tagged
+
+
+@tagged("-at_install", "post_install")
+class TestAccountPaymentPurchase(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestAccountPaymentPurchase, cls).setUpClass()
```

Comment:
Apply PEP 3135 (28-Apr-2007) to "super()."

Reply:
Done